proto/documents/v3alpha/documents.proto

```diff
@@ -218,20 +371,57 @@ message DocumentListItem {
   // User-defined metadata attributes of the document.
   map<string, string> metadata = 3;
 
-  // Every author ID who has modified this document's version.
+  // Every author ID who has ever made changes to this document.
   repeated string authors = 4;
 
   // Time when the document was created.
   google.protobuf.Timestamp create_time = 5;
 
   // Time when the document was updated.
+  // I.e. timestamp of the most recent change.
   google.protobuf.Timestamp update_time = 6;
 
   // ID of the genesis change of this document.
   string genesis = 8;
 
   // Current version of the document.
   string version = 9;
+
+  // Breadcrumbs of this document.
+  // The number of items in this list corresponds to the number of segments
+  // in the path of the document minus 1 (the document itself).
+  //
+  // E.g. for a document "/cars/jp/honda" there will be 2 breadcrumbs: "cars" and "jp".
+  //
+  // If we don't have a document for some of the path segements for whatever reason,
+  // the corresponding breadcrumb segment will be null.
+  repeated Breadcrumb breadcrumbs = 10;
+
+  // Summary of the activity on the document.
+  ActivitySummary activity_summary = 11;
+}
+
+// Summary of the activity on a document or account.
+message ActivitySummary {
+  // ID of the most recent comment.
+  string latest_comment_id = 1;
+
+  // Timestamp of the most recent comment.
+  google.protobuf.Timestamp latest_comment_time = 2;
+
+  // Total number of comments.
+  int32 comment_count = 3;
+
+  // Timestamp of the most recent change.
```

Comment:
low pri: should maybe have the id of the latest change so we can look up what exactly changed

Reply:
I can get the change details by looking directly at the CID data

And I would use it right away. Most important actually is to know **who** made the most recent change (plus the time)